### PR TITLE
🔨 Add some peds & vehicles functions

### PR DESCRIPTION
**Pull request by dezel-2109**
Hello,
Here is some functions for peds and vehichles!
The names are not the best so if you want to change them, let's go.

### CODE REVIEW

File: src/api/vehicles/client/unlockVehicle.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---unlockVehicle
+---@param vehicleId number
+---@public
+function API_Vehicles:unlockVehicle(vehicleId)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+        SetVehicleDoorsLocked(vehicleId, 1)
```
Comment (CsokiHUN) [inline src/api/vehicles/client/unlockVehicle.lua:6]:
```suggestion
    return SetVehicleDoorsLocked(vehicleId, 1)
```

File: src/api/vehicles/client/openDoor.lua [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---openDoor
+---@param vehicleId number
+---@param doorId number
+---@param canBeClosed boolean
+---@param instantly boolean
+---@public
+function API_Vehicles:openDoor(vehicleId, doorId, canBeClosed, instantly)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+        SetVehicleDoorOpen(vehicleId, doorId, canBeClosed or false, instantly or false)
```
Comment (CsokiHUN) [inline src/api/vehicles/client/openDoor.lua:9]:
```suggestion
    return SetVehicleDoorOpen(vehicleId, doorId, canBeClosed or false, instantly or false)
```

File: src/api/vehicles/client/lockVehicle.lua [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---lockVehicle
+---@param vehicleId number
+---@public
+function API_Vehicles:lockVehicle(vehicleId)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+        SetVehicleDoorsLocked(vehicleId, 2)
```
Comment (CsokiHUN) [inline src/api/vehicles/client/lockVehicle.lua:6]:
```suggestion
    return SetVehicleDoorsLocked(vehicleId, 2)
```

File: src/api/vehicles/client/openAllDoor.lua [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---openAllDoor
+---@param vehicleId number
+---@param canBeClosed boolean
+---@param instantly boolean
+---@public
+function API_Vehicles:openAllDoor(vehicleId, canBeClosed, instantly)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+        for i = 0, 5 do
+            SetVehicleDoorOpen(vehicleId, i, canBeClosed or false, instantly or false)
+        end
```
Comment (CsokiHUN) [inline src/api/vehicles/client/openAllDoor.lua:10]:
```suggestion
    end
    return
```

**Review by CsokiHUN — CHANGES_REQUESTED** [review]

**Comment by dezel-2109** [comment]
> i suggested important fixes please check it
thx ! I do it now

File: src/api/vehicles/client/openAllDoor.lua [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+---@param vehicleId number
+---@param canBeClosed boolean
+---@param instantly boolean
+---@public
+function API_Vehicles:openAllDoor(vehicleId, canBeClosed, instantly)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+        for i = 0, 5 do
+            SetVehicleDoorOpen(vehicleId, i, canBeClosed or false, instantly or false)
+    end
+    return
```
Comment (MoskalykA) [inline src/api/vehicles/client/openAllDoor.lua:11]:
```suggestion
        for i = 0, 5 do
            SetVehicleDoorOpen(vehicleId, i, canBeClosed or false, instantly or false)
        end
    return
```

File: src/api/vehicles/client/openDoor.lua [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---openDoor
+---@param vehicleId number
+---@param doorId number
+---@param canBeClosed boolean
+---@param instantly boolean
+---@public
+function API_Vehicles:openDoor(vehicleId, doorId, canBeClosed, instantly)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+    return SetVehicleDoorOpen(vehicleId, doorId, canBeClosed or false, instantly or false)
```
Comment (MoskalykA) [inline src/api/vehicles/client/openDoor.lua:9]:
```suggestion
        return SetVehicleDoorOpen(vehicleId, doorId, canBeClosed or false, instantly or false)
```

File: src/api/vehicles/client/unlockVehicle.lua [hunk 7]
```diff
@@ -0,0 +1,9 @@
+---unlockVehicle
+---@param vehicleId number
+---@public
+function API_Vehicles:unlockVehicle(vehicleId)
+    if (vehicleId) and (DoesEntityExist(vehicleId)) then
+    return SetVehicleDoorsLocked(vehicleId, 1)
```
Comment (MoskalykA) [inline src/api/vehicles/client/unlockVehicle.lua:6]:
```suggestion
        return SetVehicleDoorsLocked(vehicleId, 1)
```

File: src/api/vehicles/client/unlockVehicle.lua [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---unlockVehicle
+---@param vehicleId number
+---@public
```
Comment (MoskalykA) [inline src/api/vehicles/client/unlockVehicle.lua:3]:
```suggestion
---unlockVehicle
---@param vehicleId number
---@return void
---@public
```

File: src/api/ped/client/setIntoVehicle.lua [hunk 9]
```diff
@@ -0,0 +1,11 @@
+---setIntoVehicle
+---@param pedId number
+---@param vehicle number
+---@public
```
Comment (MoskalykA) [inline src/api/ped/client/setIntoVehicle.lua:4]:
```suggestion
---setIntoVehicle
---@param pedId number
---@param vehicle number
---@return void
---@public
```

File: src/api/ped/client/setHealth.lua [hunk 10]
```diff
@@ -0,0 +1,11 @@
+---setHealth
+---@param pedId number
+---@param value number
+---@public
```
Comment (MoskalykA) [inline src/api/ped/client/setHealth.lua:4]:
```suggestion
---setHealth
---@param pedId number
---@param value number
---@return void
---@public
```

File: src/api/ped/client/setAlhpa.lua [hunk 11]
```diff
@@ -0,0 +1,11 @@
+---setAlpha
+---@param pedId number
+---@param value number
+---@public
```
Comment (MoskalykA) [inline src/api/ped/client/setAlhpa.lua:4]:
```suggestion
---setAlpha
---@param pedId number
---@param value number
---@return void
---@public
```

File: src/api/ped/client/resetAlhpa.lua [hunk 12]
```diff
@@ -0,0 +1,10 @@
+---resetAlpha
+---@param pedId number
+---@public
```
Comment (MoskalykA) [inline src/api/ped/client/resetAlhpa.lua:3]:
```suggestion
---resetAlpha
---@param pedId number
---@return void
---@public
```

File: src/api/ped/client/leaveVehicle.lua [hunk 13]
```diff
@@ -0,0 +1,11 @@
+---leaveVehicle
+---@param pedId number
+---@param instant boolean
+---@public
```
Comment (MoskalykA) [inline src/api/ped/client/leaveVehicle.lua:4]:
```suggestion
---leaveVehicle
---@param pedId number
---@param instant boolean
---@return void
---@public
```

File: src/api/ped/client/kill.lua [hunk 14]
```diff
@@ -0,0 +1,10 @@
+---kill
+---@param pedId number
+---@public
```
Comment (MoskalykA) [inline src/api/ped/client/kill.lua:3]:
```suggestion
---kill
---@param pedId number
---@return void
---@public
```

**Review by MoskalykA — CHANGES_REQUESTED** [review]

File: src/api/ped/client/isDead.lua [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+function API_Ped:isDead(pedId)
+    if (not (DoesEntityExist(pedId))) then
+        return _NCS:die("Target ped does not exists")
+    end
+
+    if (API_Ped:getHealth(pedId) <= 0) then
+        return (true)
+    end
+
+    return (false)
```
[the remark this answers is not in the document]
Comment (CsokiHUN) [reply src/api/ped/client/isDead.lua:14]: Yes, that's really better

**Review by CsokiHUN — COMMENTED** [review]
I checked, its okay

**Review by MoskalykA — APPROVED** [review]

**Pull request merged**